bleak/backends/bluezdbus/client.py

```diff
@@ -605,17 +605,17 @@ async def write_gatt_descriptor(self, handle: int, data: bytearray) -> None:
     async def start_notify(
         self,
         char_specifier: Union[BleakGATTCharacteristicBlueZDBus, int, str, uuid.UUID],
-        callback: Callable[[str, Any], Any],
+        callback: Callable[[int, bytearray], None],
         **kwargs
     ) -> None:
         """Activate notifications/indications on a characteristic.
 
-        Callbacks must accept two inputs. The first will be a uuid string
-        object and the second will be a bytearray.
+        Callbacks must accept two inputs. The first will be a integer handle of the characteristic generating the
+        data and the second will be a ``bytearray``.
 
         .. code-block:: python
 
-            def callback(sender, data):
+            def callback(sender: int, data: bytearray):
                 print(f"{sender}: {data}")
```

Comment:
might want to change this to use logger instead of `print` while we are touching this code

Reply:
same comment applies to other backends

Reply:
I agree in general, but it then requires importing `logging` in the example code. I will push this potential documentation improvement forward to #266.